### DISCUSSION TITLE
publier/dépuplier une étiquette

### DISCUSSION THREAD

**Discussion by alexandre-bouzan** [GeotrekCE/Geotrek-admin#3529] [Ideas]
via une une case à cocher/décocher sur l'Admin-Site, rendre possible pour l'administrateur de publier ou dépublier une étiquette de Geotrek-Rando. L'idée est de pouvoir gérer facilement de l'info saisonnière ou ponctuelle à destination des usagers. 
Exemples: période de vigilance incendie, chantiers de brûlage dirigés, etc...

**Comment by babastienne** [comment]
@camillemonchicourt un avis ? Je suis plutôt ok avec l'idée.

**Comment by camillemonchicourt** [comment]
Oui OK pour moi.
Cela induit que si une étiquette est associée à une rando ou site Outdoor mais non publiée alors elle ne remonte pas dans l'API sur la route de l'objet (/trek/id par exemple).

**Reply by babastienne** [reply]
Oui tout à fait, c'est ça

**Reply by babastienne** [reply]
Je propose de ne pas gérer la notion de publication par portail pour les étiquettes cependant, ça me parait trop lourd pour un faible pluevalue.

**Reply by camillemonchicourt** [reply]
Oui en effet.
Par contre dans Geotrek-admin on affiche toutes les étiquettes, qu'elles soient publiées ou non.

**Reply by babastienne** [reply]
Oui bien entendu

**Comment by babastienne** [comment]
Available in release [2.100.0](https://github.com/GeotrekCE/Geotrek-admin/releases/tag/2.100.0)

![image](https://github.com/GeotrekCE/Geotrek-admin/assets/16348048/46a11210-98c9-4a5b-97a4-156c4fefe6ff)
